staging/kos/pkg/controllers/workqueue_prometheus/connect.go

```diff
@@ -30,8 +30,9 @@ import (
 // Package prometheus sets the workqueue DefaultMetricsFactory to produce
 // prometheus metrics. To use this package, you just have to import it.
 
-// Metrics subsystem and keys used by the workqueue.
+// Metrics namespace, subsystem and keys used by the workqueue.
```

Comment:
Ugh, I do not like not having an exact copy.  OTOH, the upstream source oddly lacks the Prometheus metrics namespace.  I guess there is an issue to raise upstream before we can do better here.